types/pdfmake/interfaces.d.ts

```diff
@@ -79,8 +79,8 @@ export interface CustomTableLayout {
     hLineStyle?: DynamicLayout<LineStyle>;
     vLineStyle?: DynamicLayout<LineStyle>;
     fillColor?: string | DynamicLayout<string>;
-    paddingLeft?: number | DynamicLayout<number>;
-    paddingRight?: number | DynamicLayout<number>;
+    paddingLeft?: DynamicLayout<number>;
+    paddingRight?: DynamicLayout<number>;
```

Comment:
I haven't been able to use custom table layouts at all in the playground. Why did you change this?

Reply:
I tested it in my env, they both generates exceptions like `layout.paddingLeft is not a function`

Reply:
Then that means other layout paramters might be erroneous.
Could you try these layouts as well? This might give us some better information.
```typescript
const customTableLayouts: CustomTableLayout[] = [
    {},
    { hLineWidth: () => 50 },
    { vLineWidth: () => 50 },
    { hLineColor: () => '#ff0000' },
    { hLineColor: '#ff0000' },
    { vLineColor: () => '#ff0000' },
    { vLineColor: '#ff0000' },
    { hLineStyle: () => ({ dash: { length: 10, space: 5 }}) },
    { vLineStyle: () => ({ dash: { length: 10, space: 5 }}) },
    { fillColor: () => '#ff0000' },
    { fillColor: '#ff0000' },
    { paddingLeft: () => 50 },
    { paddingRight: () => 50 },
    { paddingTop: () => 50 },
    { paddingBottom: () => 50 },
    { fillOpacity: () => 50 },
    { fillOpacity: 50 },
    { defaultBorder: true }
];
```typescript

Reply:
@jeralm all the layout settings you mentioned seem to work fine !